examples/tv-app/linux/include/tv-channel/TvChannelManager.cpp

```diff
@@ -47,24 +47,6 @@ CHIP_ERROR TvChannelManager::Init()
     return err;
 }
 
-EmberAfTvChannelInfo TvChannelManager::proxyChangeChannelRequest(std::string match)
-{
-    // TODO: Insert code here
-    EmberAfTvChannelInfo channel = {};
-    return channel;
-}
-
-bool TvChannelManager::proxyChangeChannelByNumberRequest(uint16_t majorNumer, uint16_t minorNumber)
-{
-    // TODO: Insert code here
-    return true;
-}
-bool TvChannelManager::proxySkipChannelRequest(uint16_t count)
-{
-    // TODO: Insert code here
-    return true;
-}
-
 std::vector<EmberAfTvChannelInfo> TvChannelManager::proxyGetTvChannelList()
```

Comment:
Preexisting but this function is also returning pointers-to-stack...

Reply:
Created an [issue](https://github.com/project-chip/connectedhomeip/issues/8087) to resolve this.